firebase-debug.log

```diff
@@ -0,0 +1,25 @@
+[debug] [2019-02-06T08:26:06.799Z] ----------------------------------------------------------------------
```

Comment:
I am not sure if this is necessary in github. If it is, then ignore my comment. If not, then I would add firebase-debug.log to .gitignore.